src/coreclr/jit/emitarm64.h

```diff
@@ -1012,6 +1022,17 @@ inline static bool insScalableOptsNone(insScalableOpts sopt)
     return sopt == INS_SCALABLE_OPTS_NONE;
 }
 
+inline static bool insScalableOptsWithPredicatePair(insScalableOpts sopt)
```

Comment:
looks like we missed the summary header for newer SVE methods we added recently. do you mind adding those?